R/getPrevalence.R

```diff
@@ -580,18 +586,11 @@ setMethod("agglomerateByPrevalence", signature = c(x = "SummarizedExperiment"),
             if(!is.null(rank)){
                 rowData(other_x)[,rank] <- other_label
             }
-            # temporary fix until TSE supports rbind
-            class <- c("SingleCellExperiment","RangedSummarizedExperiment",
-                       "SummarizedExperiment")
-            class_x <- class(x)
-            if(!(class_x %in% class)){
-                class <- "SingleCellExperiment"
-            } else {
-                class <- class[class_x == class]
-            }
-            x <- rbind(as(x[f,],class),
-                       as(other_x,class))
-            x <- as(x,class_x)
+            x <- rbind(x[f,], other_x)
+        }
+        if (agglomerate.tree == TRUE){
```

Comment:
`if( agglomerate.tree )`

Reply:
This was not addressed. As agglomerate.tree is already boolean, it is good practice to use it directly instead of comparing boolean value to boolean value